ci/prebuilt/BUILD

```diff
@@ -36,6 +36,16 @@ cc_library(
 )
 
 cc_library(
+    name = "fmtlib",
+    hdrs = glob([
+        "thirdparty/fmtlib/fmt/*.cc",
```

Comment:
Did you determine if this is necessary still?

Reply:
Trying to do a regular local bazel build was failing, but adding the *.cc back in fixed it